source/fundamentals/data-formats/documents.txt

```diff
@@ -284,6 +289,96 @@ For more information on retrieving and manipulating MongoDB data, see our
 For more information on this class, see the
 :java-docs:`BsonDocument API documentation </apidocs/bson/org/bson/BsonDocument.html>`.
 
+JsonObject
+-------------
+
+The ``JsonObject`` class acts as a wrapper for JSON strings.
+If you want to work primarily with JSON strings in your application, 
+use the ``JsonObject`` class. For applications that work with JSON
+strings, ``JsonObject`` is a more performant alternative to both ``Document``
+and ``BsonDocument`` as ``JsonObject`` does not create an intermediate
+``Map`` object.
+
+By default, ``JsonObject`` stores :manual:`Extended JSON </reference/mongodb-extended-json/>`.
+You can customize the format of JSON in ``JsonObject`` by specyfying a
```

Comment:
```suggestion
You can customize the format of JSON in ``JsonObject`` by specifying a
```

Reply:
Good catch. I will use a use a spell checking tool on my next PR. Thanks for spotting this.